react/features/participants-pane/selectors.js

```diff
@@ -0,0 +1,108 @@
+import { MEDIA_TYPE } from '../base/media';
```

Comment:
I think we usually have selectors in functions.js. Can you please check if that is the usual case so we keep that consistency?